autofill/autofill-impl/src/main/res/values-ro/strings-autofill-impl.xml

```diff
@@ -143,4 +143,23 @@
     <string name="credentialManagementClearNeverForThisSiteDialogPositiveButton">Resetează</string>
     <string name="credentialManagementClearNeverForThisSiteDialogNegativeButton">Anulare</string>
 
+    <string name="credentialManagementDeleteAllPasswordsMenu">Șterge toate parolele</string>
+    <plurals name="credentialManagementDeleteAllPasswordsConfirmationTitle" instruction="Placeholder is a number representing how many passwords will be deleted">
+        <item quantity="one">Sigur dorești să ștergi %1$d parolă?</item>
+        <item quantity="few">Sigur dorești să ștergi %1$d parole?</item>
+        <item quantity="other">Sigur dorești să ștergi această %1$d de parole?</item>
```

Comment:
This doesn't make sense. But on the other hand, I don't think "other" will be ever triggered in this scenario.